inlong-dataproxy/dataproxy-source/pom.xml

```diff
@@ -46,19 +46,13 @@
 		<dependency>
 			<groupId>org.powermock</groupId>
 			<artifactId>powermock-module-junit4</artifactId>
-			<version>2.0.2</version>
+			<version>${powermock.version}</version>
```

Comment:
The third-party dependent version needs to be external to the parent pom  <dependencymanagement> file

Reply:
dependencymanagement

Reply:
ok, move dependency to pom.xml of inlong-dataproxy